### ISSUE TITLE
Neighborhoods task

### ISSUE THREAD

**Issue by reinvantveer** [reinvantveer/Topology-Learning#26]
- [x] Add recurrent dropout to prevent overfitting
- [x] Try Normalizer, StandardScaler, MinMaxScaler and RobustScaler
- [x] Create RBF kernel model
- [x] Run models with test data, print result
- [x] Create Linear model
- [x] Create Polynomial model
- [x] Create k-nn model
- [x] Create decision tree model
- [x] Create scatterplot area/number of inhabitants
- [x] Create RNN model
- [x] Try to re-run RNN model with 256 instead of 512 points per geom
- [x] Re-run with softmax activated layer, note down results
- [x] Re-run with sigmoid activated layer (single node output), note down results
- [x] Create and run logistic regression model
- [x] Re-run baselines model with grid search on grid centered on best results 
- [x] Run best configuration 10 times, note down standard deviation
